tools/setup/emoji/zulip_emoji_map.py

```diff
@@ -0,0 +1,1505 @@
+from typing import Any, Dict
+
+ZULIP_EMOJI_MAP = {
```

Comment:
should this be called EMOJI_NAMES?
Potentially this file also should be emoji_names.py?

Reply:
I like `zulip_emoji_map`(lowercase) more since that clearly conveys that it is a mapping not a list.

Reply:
hm, possibly I think each entry should indeed just be a list of names :). I haven't read the code that interacts with this, though.

I think in general it's not that useful to put type information in the variable names though, especially now that we have mypy. E.g. I think it'll be pretty clear from the code like the following what is going on:
```
names = emoji_names[codepoint]
// do something with names['canonical_names'], or names['aliases']
```

Reply:
This is a big renaming change if we rename it, so we can wait for Tim to comment :).

Reply:
Talked to Tim, let's go with `emoji_name_maps` for now. Though I suspect we might actually want to make this emoji_name_lists and just have them be lists :).